src/System/Status/StatusChecker.php

```diff
@@ -389,11 +389,14 @@ public static function getCASStatus($public_only = true): array
         if ($status === null) {
             $status['status'] = self::STATUS_NO_DATA;
             if (!empty($CFG_GLPI['cas_host'])) {
-                $url = $CFG_GLPI['cas_host'];
+                // Rebuild CAS URL
+                // see `CAS_Client::_getServerBaseURL()`
+                $url = 'https://' . $CFG_GLPI['cas_host'];
```

Comment:
If `http` is now allowed, how do you know if it is `http` or `https` here ?

Reply:
A CAS server is always exposed on HTTPS.